src/main/java/frc/robot/robotcontainers/BeetleContainer.java

```diff
@@ -22,19 +23,26 @@ public class BeetleContainer extends RobotContainer {
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public BeetleContainer() {
     // Configure the button bindings
-    configureButtonBindings();
+        configureButtonBindings();
   }
 
+private void configureButtonBindings(){
+
+  CommandXboxController driver = new CommandXboxController(0);
+  CommandXboxController operator = new CommandXboxController(1);
+  driver.y().whileTrue(new ClimberRPM(-0.1));
```

Comment:
Shouldn't add the climber here, since Beetle does not have a climber.

Reply:
We will test this on Beetle with prototype climber.